charts/springboot/templates/ingress-external.yaml

```diff
@@ -15,22 +15,31 @@ metadata:
     {{ required "'project' value not found" .Values.project }}-app: "true"
   annotations:
     {{- range $key, $value := .Values.ingressExternal.annotations }}
-      {{ $key }}: {{ $value | quote }}
+    {{ $key }}: {{ $value | quote }}
     {{- end }}
```

Comment:
Restore formating - this is inside loop statment would be good if it was idented?

Reply:
Previous indentation was bad and probably was not tested. It caused invalid YAML so I have fixed it.